public/scripts/views/repoView.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+(function(module) {
+  const repoView = {};
+
+  const ui = function() {
+    let $github = $('#github');
+    $github.find('ul').empty();
+    //$github.show().siblings().hide(); // QUESTION: what's this doing?
+  };
```

Comment:
That's from the lab assignment set up, where the gitHub repos were listed on the about page. So, it's only showing the repo data, not other things. You are handling that in your controllers, right?